### PR TITLE
Add core protos Artman configs for Java

### PR DESCRIPTION
**Pull request by shinfan**
Toolkit PR: https://github.com/googleapis/toolkit/pull/1133
Artman PR: googleapis/artman#143

### CODE REVIEW

File: gapic/packaging/dependencies.yaml [hunk 1]
```diff
@@ -73,6 +79,9 @@ google-common-protos_version:
   ruby:
     name_override: googleapis-common-protos
     lower: '1.3.1'
+  java:
+    name_override: grpc-google-common-protos
+    lower: '0.1.5'
```
Comment (garrettjonesgoogle) [inline gapic/packaging/dependencies.yaml]: Why not 0.1.6?

File: gapic/packaging/dependencies.yaml [hunk 2]
```diff
@@ -82,3 +91,7 @@ google-iam-v1_version:
   ruby:
     name_override: grpc-google-iam
     lower: '0.6.8'
+
+api_common_version:
+  java:
+    lower: '0.0.2'
```
Comment (garrettjonesgoogle) [inline gapic/packaging/dependencies.yaml]: add newline

Comment (shinfan) [inline gapic/packaging/dependencies.yaml]: Done

File: gapic/packaging/dependencies.yaml (hunk 1 above)
Comment (shinfan) [inline gapic/packaging/dependencies.yaml]: Done

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**